### DISCUSSION TITLE
Handle special tags like Kustomize releases?

### DISCUSSION THREAD

**Discussion by tboerger** [renovatebot/renovate#12179] [Request Help] [answered]
I've built a docker image which installs the Kustomize binary. But the tags got a pretty special tagging/release. If you look at https://github.com/kubernetes-sigs/kustomize/releases/tag/kustomize%2Fv4.4.0 they are releasing different stuff within the same repository.

So how can I configure Renovate to update the dependency automatically? Generally I'm updating the dependencies with something similar to this:

### Dockerfile

```
# renovate: datasource=github-releases depName=helm/helm
ENV HELM_VERSION=3.7.1
```

###  renovate.json

```
    "regexManagers": [{
        "fileMatch": [
            "Dockerfile"
        ],
        "matchStrings": [
            "# renovate: datasource=(?<datasource>\\S+) depName=(?<depName>\\S+)( versioning=(?<versioning>.*?))?\nENV .*?_VERSION=(?<currentValue>.*)"
        ],
        "versioningTemplate": "{{#if versioning}}{{{versioning}}}{{else}}semver{{/if}}"
    }],
```

**Comment by viceice** [comment, marked as answer]
I'm doing this:
https://github.com/visualon/docker-images/blob/e48904fe1d07e7244198a586c7e1943102e2d915/.github/renovate.json#L72-L75

```json
{
  "packageRules": [
    {
      "packageNames": ["kubernetes-sigs/kustomize"],
      "extractVersion": "^kustomize\\/v(?<version>\\d+\\.\\d+\\.\\d+)$"
    },
  ]
}
```


```Dockerfile
FROM ghcr.io/visualon/buildpack:latest@sha256:b0d5782f1c090af1226c94b6bad9082e9c5af05f8dd8ea25cc3aed27b6c5e3a7

# renovate: datasource=github-releases lookupName=kubernetes-sigs/kustomize
RUN install-tool kustomize 4.0.4
```

**Reply by tboerger** [reply]
Great, thanks for the fast response!
